perf/micro/current-thread-scheduler/observable/throw.js

```diff
@@ -0,0 +1,20 @@
+var RxOld = require("rx");
+var RxNew = require("../../../../index");
+
+module.exports = function _throw(suite) {
```

Comment:
'_' was chosen to avoid collision to language reserved keywords, would be appreciate to let me know if this is acceptable styles. Otherwise, will change it accordingly.